cheddar/cheddar-server/build.gradle

```diff
@@ -12,4 +10,5 @@ dependencies {
     compile "org.glassfish.jersey.ext:jersey-spring3:${jerseyVersion}"
     compile "org.glassfish.jersey.containers:jersey-container-grizzly2-http:${jerseyVersion}"
     compile 'javax.servlet:javax.servlet-api:3.1.0'
+    compile "org.slf4j:jul-to-slf4j:${slf4jVersion}"
```

Comment:
There's a corresponding removal of this dependency in `logging-api.gradle`. This particular dependency is only needed in this subproject.